.github/workflows/verify-generation-config.yaml

```diff
@@ -18,6 +48,18 @@ jobs:
           docker volume rm repo-google-cloud-java
         fi
         docker volume create --name "repo-google-cloud-java" --opt "type=none" --opt "device=$(pwd)" --opt "o=bind"
+    - name: update googleapis commit to latest
+      id: update-commit
+      shell: bash
+      run: |
+        mkdir tmp-googleapis
+        git clone https://github.com/googleapis/googleapis.git tmp-googleapis
```

Comment:
Maybe we can do a shallow copy of the repo since we only want the committish?

Reply:
Done, using partial clone.